docker/images/es-0.90/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM java:8-jre
+
+RUN apt-key adv --keyserver ha.pool.sks-keyservers.net --recv-keys 46095ACC8548582C1A2699A9D27D666CD88E42B4
+
+ENV ELASTICSEARCH_MAJOR 0.90
+ENV ELASTICSEARCH_VERSION 0.90.13
+ENV ELASTICSEARCH_REPO_BASE http://packages.elasticsearch.org/elasticsearch/0.90/debian
+
+RUN echo "deb $ELASTICSEARCH_REPO_BASE stable main" > /etc/apt/sources.list.d/elasticsearch.list && \
+     apt-get update && \
+     apt-get install -y --no-install-recommends elasticsearch=$ELASTICSEARCH_VERSION && \
+     rm -rf /var/lib/apt/lists/*
+
+ENV PATH /usr/share/elasticsearch/bin:$PATH
+
+WORKDIR /usr/share/elasticsearch
+
+COPY config ./config
+
```

Comment:
Can you add a comment here? Something like: "Create ES tree and set permissions"